vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -623,3 +621,9 @@ private List getContainerEnvs(config, imageName, defaultEnvVars, defaultConfig)
 
     return containerEnv
 }
+
+private void invalidateStash(def config, String prefix) {
```

Comment:
Would it be useful to have the `invalidateStash` method in `com.sap.piper.Utils`?
We already have `stash`, `unstash`, `unstashAll`,...

Reply:
I extracted it simply to a method in order to reduce code-duplication.
Other question would be: is that pattern of invalidating stashes use somewhere else? I'm not aware.
Beside that is is simply a call to `stash` together with a log message.